aspnetcore/fundamentals/http-logging/samples/8.x/SampleHttpLoggingInterceptor.cs

```diff
@@ -0,0 +1,85 @@
+using Microsoft.AspNetCore.HttpLogging;
+
+namespace HttpLoggingSample;
+
+internal sealed class SampleHttpLoggingInterceptor : IHttpLoggingInterceptor
+{
+    public ValueTask OnRequestAsync(HttpLoggingInterceptorContext logContext)
+    {
+        if (!logContext.HttpContext.Request.Path.StartsWithSegments("/api"))
```

Comment:
Sorry, I know this was copied from my own sample, but directly matching the path like this is a bad habit we don't want to encourage. Can you change this to exclude logs for POST requests instead?